e2e/e2e-values.yml

```diff
@@ -1,12 +1,16 @@
 verboseLog: true
-image:
+forwarderImage:
+  pullPolicy: IfNotPresent
+  tag: 1.20.5
+  repository: newrelic/k8s-events-forwarder
+agentImage:
   pullPolicy: Never
   tag: e2e
   repository: e2e/newrelic-infrastructure
```

Comment:
Why do we have two agents?

Reply:
Since the "forwarderImage" has the "NRIA_IS_FORWARD_ONLY" flag and it is merely forwarding KSM data
The "agentImage" is creating the entity, monitoring the underlying host, forwarding kubelet and control plane data

Reply:
Got it, for a moment I forgot both "agent flavours" belonging to the same chart and actually being different images.